go.mod

```diff
@@ -378,7 +378,7 @@ replace (
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 	github.com/golang-jwt/jwt/v4 => github.com/golang-jwt/jwt/v4 v4.2.0
 	github.com/golang/freetype => github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0
-	github.com/golang/glog => github.com/golang/glog v1.0.0
```

Comment:
this looks spurious, we don't want to backlevel this, right?

Reply:
Running `hack/lint-dependencies.sh` immediately after `hack/pin-dependency.sh github.com/google/cel-go v0.12.5` gives me the following, based off of d581cc90adba6c84919738841fe3e07302d53e33:
```
Check all unwanted dependencies and update its status.
These modules are pinned to versions different than the minimal preferred version.
That means that without replace directives, a different version would be selected,
which breaks consumers of our published modules.
1. Use hack/pin-dependency.sh to switch to the preferred version for each module
2. Run hack/update-vendor.sh to rebuild the vendor directory
3. Run hack/lint-dependencies.sh to verify no additional changes are required

github.com/golang/glog
    pinned:    v1.0.0
    preferred: v0.0.0-20160126235308-23def4e6c14b
    hack/pin-dependency.sh github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
All pinned versions of checked dependencies match their preferred version.
```

Not 100% sure why glog is getting changed here - I don't think it's a direct dependency of cel-go, at least.

Reply:
hmm, looks like it was in 0.12.4:

`github.com/google/cel-go@v0.12.4 github.com/golang/glog@v1.0.0`

it doesn't really matter, since we don't actually vendor it